editor/app/layout.tsx

```diff
@@ -33,6 +34,7 @@ export default function RootLayout({
     <html lang="en" style={{ fontSize: "14px" }}>
       <AuthProvider>
         <body>
+          <EnvironmentBanner />
```

Comment:
i noticed that the extra height of the banner is causing a second scroll in the sidebar. more of a product question for @johnclary. wanted to mention since we have a similar layout shift in Moped from env banners that has caused some confusion at times.

https://github.com/user-attachments/assets/348dcba6-a25e-49b2-a595-5bcfd8ea4f00

Reply:
hmm, yeah the double scrollbar is blech. @mateoclarke i think we need to get the banner into the flex layout somehow. if it's helpful, i made [this visual](https://github.com/cityofaustin/vision-zero/pull/1686#discussion_r1941742376) the last time i reworked the containers.